### PR TITLE
Add duoshao.app v0.1.7

### PR DESCRIPTION
**Pull request by wangkezun**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/duoshao.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "duoshao" do
+  version "0.1.7"
+  sha256 "a34416bbe04834868a267683c16101c2999949d56b4ee7cc30958e6667e1c382"
+
+  # haokuai.cdn.tinyservices.net was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/duoshao.rb:5]: Please change this to:
```
# haokuai.cdn.tinyservices.net/duoshao/ was verified as official when first introduced to the cask
```

File: Casks/duoshao.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  url "https://haokuai.cdn.tinyservices.net/duoshao/Duoshao-#{version}.dmg"
```
Comment (miccal) [inline Casks/duoshao.rb:6]: Please add the following `appcast`:
```
appcast "https://electron-releases-1259797650.cos.ap-nanjing.myqcloud.com/duoshao/latest-mac.yml"
```
I am not certain if the `1259797650` string changes, though.

File: Casks/duoshao.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  name "duoshao"
+  desc "多少记账"
```
Comment (miccal) [inline Casks/duoshao.rb:8]: Please change this to an English version of the `desc`.

File: Casks/duoshao.rb [hunk 4]
```diff
@@ -2,10 +2,10 @@
   version "0.1.7"
   sha256 "a34416bbe04834868a267683c16101c2999949d56b4ee7cc30958e6667e1c382"
 
-  # haokuai.cdn.tinyservices.net was verified as official when first introduced to the cask
+  # haokuai.cdn.tinyservices.net/duoshao/ was verified as official when first introduced to the cask
   url "https://haokuai.cdn.tinyservices.net/duoshao/Duoshao-#{version}.dmg"
   name "duoshao"
-  desc "多少记账"
+  desc "duoshao"
```
Comment (vitorgalvao) [inline Casks/duoshao.rb:9]: This is not a description, it’s the name of the software. It needs to explain what this is or does.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**